components/HomePage/SponsorHighlight.js

```diff
@@ -99,9 +102,10 @@ const SponsorHighlight = ({ className, eventSlug }) => {
               We partner with industry leading organizations and companies that
               are looking to connect with software developers and practitioners.
             </p>
-            <ActionButton
```

Comment:
Renaming is optional, but it seemed like "ActionButton" was being used several places without consistent meaning.